modROS.lua

```diff
@@ -286,6 +286,14 @@ function ModROS:publish_laser_scan_func()
     local cos = math.cos
     local sin = math.sin
     local LS_FOV = mod_config.laser_scan.angle_max - mod_config.laser_scan.angle_min
+    -- adding the first laser scan frame to raycastNode (x left, y up, z into the page)
+    local laser_scan_frame_1 = createTransformGroup("laser_scan_frame_1")
+    link(self.instance_veh.cameraNode, laser_scan_frame_1)
+    -- apply a transfrom to laser_scan_frame_1
+    local x, y, z = 0, 0, 0
+    local rot_x, rot_y, rot_z = -math.pi/6, 0, 0
```

Comment:
Where does the `math.pi/6` come from?

Reply:
it's the config setting

Reply:
to make the laser frame tilted math.pi/6. I've made this setting configurable in cc44abb350b29f5d925c24f2e26dd6e9f20a30b7